runtime/local/src/precompiles.rs

```diff
@@ -23,7 +24,7 @@ impl<R> LocalNetworkPrecompiles<R> {
     /// Return all addresses that contain precompiles. This can be used to populate dummy code
     /// under the precompile.
     pub fn used_addresses<AccountId: From<H160>>() -> impl Iterator<Item = AccountId> {
-        sp_std::vec![1, 2, 3, 4, 5, 6, 7, 8, 1024, 1025, 1026]
+        sp_std::vec![1, 2, 3, 4, 5, 6, 7, 8, 1024, 1025, 1026, 20481]
```

Comment:
A random question, but is there a rule for deciding the precompiled contract address?

Reply:
We have alreaday one precompile and it is stating that Astar precompiles will be using addresses 0x5000 and onwards. this is 0x5001